tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -416,6 +416,10 @@ TEST(tfw_str_eq_cstr_pos, plain)
 	TfwStr *fox = make_plain_str(foxstr), *c, *end;
 	long i, offset = 0, foxlen = fox->len;
 
+	/* unlike to tfw_str_eq_cstr_off we don't  know which pointer we should use
+	 * to address string element at 'offset' position from begging of the string.
+	 * go though chunks, manually to get needed pointers
+	 */
```

Comment:
The comment is completely unnecessary. The behaviour is implied by the nature of the tested function itself.
In this case, however, the test is against a plain string, so it doesn't make sense to go over chunks of the string as there are none.